From fbcb4ffa8648d0aa5be01c11816423a483f245ae Mon Sep 17 00:00:00 2001
From: Ying Fang <fangying1@huawei.com>
Date: Tue, 26 May 2020 22:39:23 +0800
Subject: [PATCH] hw/arm/virt: add missing compat for kvm-no-adjvtime

Machine compatibility for kvm-no-adjvtime is missed,
let's add it for virt machine 4.0

Signed-off-by: Ying Fang <fangying1@huawei.com>

diff --git a/hw/arm/virt.c b/hw/arm/virt.c
index 4c727939..133d36a4 100644
--- a/hw/arm/virt.c
+++ b/hw/arm/virt.c
@@ -2492,6 +2492,7 @@ static void virt_machine_4_0_options(MachineClass *mc)
     compat_props_add(mc->compat_props, hw_compat_4_0, hw_compat_4_0_len);
     vmc->no_ged = true;
     mc->auto_enable_numa_with_memhp = false;
+    vmc->kvm_no_adjvtime = true;
 }
 DEFINE_VIRT_MACHINE(4, 0)
 
-- 
2.23.0

